app/src/main/java/it/quip/android/partials/EndlessScrollHandler.java

```diff
@@ -0,0 +1,47 @@
+package it.quip.android.partials;
```

Comment:
Let's call it listeners, since thats what all the interfaces and abstract classes are. And listeners are an objective paradigm we can group around.